internal/license_reporting/license_reporting.go

```diff
@@ -0,0 +1,92 @@
+package licensereporting
```

Comment:
@j1m-ryan is it possible to have some sort of unit tests for this one?

Reply:
Yeah I will add some